phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -452,7 +454,14 @@ oneStatement returns [BindableStatement ret]
 finally{ contextStack.pop(); }
     
 explain_node returns [BindableStatement ret]
-    :   EXPLAIN q=oneStatement {$ret=factory.explain(q);}
+    :   EXPLAIN (w=WITH)? (r=REGIONS)? q=oneStatement
```

Comment:
Would it make sense to not change what the EXPLAIN statement generates. But have all plan info/attributes generated all the time.

And have different kinds of plan output generators. 
We may want to look at how other DBs have implemented it.

Reply:
Sounds good, let me try this tomorrow i.e. let the source generate all info in the explain plan including region info, but it's upto `ExecutableExplainStatement` to control what to print.